.github/workflows/conformance.yml

```diff
@@ -34,21 +34,25 @@ jobs:
             consul-image: "hashicorp/consul:1.12"
             envoy-image: "envoyproxy/envoy:v1.22-latest"
             consul-k8s-version: "v0.49.1"
+            envoy-field: "global.imageEnvoy"
           - name: "consul@v1.13 + consul-k8s@v0.49.1"
             api-gateway-image: "hashicorppreview/consul-api-gateway:0.5-dev"
             consul-image: "hashicorp/consul:1.13"
             envoy-image: "envoyproxy/envoy:v1.22-latest"
             consul-k8s-version: "v0.49.1"
+            envoy-field: "global.imageEnvoy"
           - name: "consul@v1.14-dev + consul-k8s@v1.0.0"
             api-gateway-image: "hashicorppreview/consul-api-gateway:0.5-dev"
             consul-image: "hashicorp/consul:1.14"
-            envoy-image: "envoyproxy/envoy:v1.22-latest"
+            envoy-image: "envoyproxy/envoy:v1.24-latest"
             consul-k8s-version: "main"
+            envoy-field: "apiGateway.imageEnvoy"
           - name: "consul@v1.15-dev + consul-k8s@v1.0.0"
             api-gateway-image: "hashicorppreview/consul-api-gateway:0.5-dev"
             consul-image: "hashicorppreview/consul:1.15-dev"
-            envoy-image: "envoyproxy/envoy:v1.22-latest"
+            envoy-image: "envoyproxy/envoy:v1.24-latest"
```

Comment:
Just FYI I don't believe we've tested on 1.24 -- I believe the helm chart is currently using 1.23.1, so this may/may not be compatible at the moment.

Reply:
Yea, my intent here was just to do a first CI test and see if it Just Works™, will revert if needed 😁

Reply:
Tested locally by walking through the Learn guide and Envoy v1.24-latest appears to be working just fine!